modules/apps/callbacks/fee_transfer_test.go

```diff
@@ -0,0 +1,217 @@
+package ibccallbacks_test
+
+import (
+	"fmt"
+
+	sdkmath "cosmossdk.io/math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	feetypes "github.com/cosmos/ibc-go/v7/modules/apps/29-fee/types"
+	"github.com/cosmos/ibc-go/v7/modules/apps/callbacks/types"
+	ibctesting "github.com/cosmos/ibc-go/v7/testing"
+)
+
+var (
+	defaultRecvFee    = sdk.Coins{sdk.Coin{Denom: sdk.DefaultBondDenom, Amount: sdkmath.NewInt(100)}}
+	defaultAckFee     = sdk.Coins{sdk.Coin{Denom: sdk.DefaultBondDenom, Amount: sdkmath.NewInt(200)}}
+	defaultTimeoutFee = sdk.Coins{sdk.Coin{Denom: sdk.DefaultBondDenom, Amount: sdkmath.NewInt(300)}}
+)
+
+func (s *CallbacksTestSuite) TestIncentivizedTransferCallbacks() {
+	testCases := []struct {
+		name            string
+		transferMemo    string
+		expCallbackType types.CallbackType
+		expSuccess      bool
+	}{
+		{
+			"success: transfer with no memo",
```

Comment:
both here and in `transfer` we have similar testCases both for this function and for their `timeout` equivalents. I'm wondering if it might make more sense to combine these.